release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -513,6 +513,10 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 ==== CoreDNS update to version 1.11.3
 
 {product-title} {product-version} now includes CoreDNS version 1.11.3.
+[id="ocp-4-17-ebpf-program-support-ingress-node-firewall-operator_{context}"]
+==== eBPF program support for Ingress Node Firewall Operator (Tech Preview)
+
+The bpfman Operator, available as a Technology Preview, allows you to deploy and manage eBPF programs. It facilitates the loading, unloading, modifying, and monitoring of eBPF programs in {product-title} clusters. For more information on deploying the bpfman Operator, see xref:../networking/network_security/bpfman-operator/bpfman-operator-about.adoc#bpfman-operator-about[About the bfpman Operator].
```

Comment:
we should mention something about security or loading eBPF in secure way ?

Reply:
+1